controllers/forwarding/forwarding_controller.go

```diff
@@ -92,7 +96,7 @@ func (r *ReconcileForwarder) Reconcile(ctx context.Context, request ctrl.Request
 			}
 		}
 		if validationerrors.IsValidationError(err) {
-			condition := logging.CondInvalid("validation failed: %v", err)
+			condition := logging.NewCondition(ValidationCondition, corev1.ConditionTrue, ValidationFailureReason, "%v", err)
```

Comment:
I generally find it confusing but  the "true/false" means the condition is currently enabled (or not).  We generally "refresh" the conditions on reconciliation but I believe it would be more obvious if the conditions were persistent and you would see them flip on and off or if we had a list of other conditions that applied for the normal use case, as described by @cahartma